host/libvirt_lxc_backend.go

```diff
@@ -669,11 +672,28 @@ func (c *libvirtContainer) watch(ready chan<- error) error {
 			g.Log(grohl.Data{"at": "get_streams", "status": "error", "err": err.Error()})
 			return err
 		}
+
 		log := c.l.openLog(c.job.ID)
 		defer log.Close()
-		// TODO: log errors from these
-		go log.Follow(1, stdout)
-		go log.Follow(2, stderr)
+
+		muxConfig := logmux.Config{
+			AppName: c.job.Metadata["flynn-controller.app"],
```

Comment:
I have [a commit to fix this](https://github.com/flynn/flynn/commit/529ff1c2fd60382d89b3ab4b72bdffa4d52605a5) on a branch, I will PR it individually soon.